plugins/org.eclipse.papyrus.uml.diagram.sequence.runtime/messages.properties

```diff
@@ -107,3 +107,6 @@ Constraintlink10CreationTool_title=Constraint link
 Constraintlink10CreationTool_desc=Create a constraint link
 ConsiderIgnoreFragment16CreationTool_title=ConsiderIgnoreFragment
 ConsiderIgnoreFragment16CreationTool_desc=Create consider Ignore Fragment
+SynchronousMessagePreferencesLabel=Synchronous Messages
+CreateExecutionAfterSyncMessageAutomaticallyLabel=Automatically create execution specification after creating a sync message?
```

Comment:
Check boxes should not be phrased as questions.  And automaticity is implied.  I would suggest

* Create execution specification for synchronous messages
* Create reply messages for synchronous calls

Note the specificity of calls in the second case because synchronous signals cannot have replies.